core/src/test/java/org/infinispan/api/mvcc/read_committed/CacheAPIPessimisticTest.java

```diff
@@ -1,17 +1,20 @@
 package org.infinispan.api.mvcc.read_committed;
 
+import org.infinispan.api.BaseCacheAPIPessimisticTest;
+import org.infinispan.api.CacheAPITest;
 import org.infinispan.configuration.cache.ConfigurationBuilder;
 import org.infinispan.transaction.LockingMode;
```

Comment:
I guess this will become an unused import - please check Analyze / Run inspection by name / Unused import in IntellliJ